### PR TITLE
Update test for updated file

### PR DESCRIPTION
**Pull request by rosteen**
HST DRZ parser test was failing due to an updated remote file, this fixes it.

### CODE REVIEW

**Review by pllim — APPROVED** [review]
LGTM. Thanks!!!

**Review by kecnry — APPROVED** [review]

**Comment by kecnry** [comment]
(still is a shape mismatch - but assuming numbers don't change significantly, approval still stands and feel free to merge!)

**Comment by rosteen** [comment]
> (still is a shape mismatch - but assuming numbers don't change significantly, approval still stands and feel free to merge!)

It passes for me locally with this shape (I tried with the one from the other failed test log first), I don't know why they're different 😵

File: jdaviz/configs/imviz/tests/test_parser.py [hunk 1]
```diff
@@ -381,11 +381,11 @@ def test_parse_hst_drz(self, imviz_helper):
         data = imviz_helper.app.data_collection[0]
         comp = data.get_component('SCI,1')
         assert data.label == 'contents[SCI,1]'  # download_file returns cache loc
-        assert data.shape == (4298, 4220)
+        assert data.shape == (4299, 4220)
```
Comment (rosteen) [inline jdaviz/configs/imviz/tests/test_parser.py:384]:
```suggestion
        assert data.shape == (4299, 4219)
```

File: jdaviz/configs/imviz/tests/test_parser.py [hunk 2]
```diff
@@ -381,11 +381,11 @@ def test_parse_hst_drz(self, imviz_helper):
...
         assert_allclose(data.meta['PHOTFLAM'], 7.8711728E-20)
         assert isinstance(data.coords, WCS)
         assert comp.units == 'electron/s'
-        assert comp.data.shape == (4298, 4220)
+        assert comp.data.shape == (4299, 4220)
```
Comment (rosteen) [inline jdaviz/configs/imviz/tests/test_parser.py:388]:
```suggestion
        assert comp.data.shape == (4299, 4219)
```

**Comment by pllim** [comment]
You might be using an older cached copy locally? 🤔

**Comment by rosteen** [comment]
> You might be using an older cached copy locally? 🤔

I mean, it pulled down a copy that at least didn't match the test, so it was updated compared to what the test had been running against at least. And now apparently all the stat functions have different values as well, maybe someone else should try running this locally.

**Comment by pllim** [comment]
For some reason, the caching can be weird. I can see the same error as CI locally. I will push a commit here.

File: jdaviz/configs/imviz/tests/test_parser.py [hunk 3]
```diff
@@ -407,29 +407,29 @@ def test_parse_hst_drz(self, imviz_helper):
         assert_allclose(sky.dec.deg, 10.802045612042956, rtol=1e-3)
         data_unit = u.electron / u.s
         assert_quantity_allclose(tbl[0]['background'], 0.0014 * data_unit)
-        assert_quantity_allclose(tbl[0]['sum'], 126.789257 * data_unit, rtol=1e-3)
+        assert_quantity_allclose(tbl[0]['sum'], 115.944737 * data_unit, rtol=1e-3)
```
Comment (pllim) [inline jdaviz/configs/imviz/tests/test_parser.py:410]: Hmm. Sum decreased and the object is rounder. Maybe they removed an artifact in the update.

**Comment by pllim** [comment]
BTW I had to force the test to run the download part with `cache="update"` once to force the local data to update.

**Comment by rosteen** [comment]
> BTW I had to force the test to run the download part with `cache="update"` once to force the local data to update.

I did a quick look through the docs to see if the remote_data option had a flag like that and didn't find it, good to know!

**Comment by pllim** [comment]
Yeah, HPC people refactored it at one point and introduced the update option. API doc at https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html#astropy.utils.data.download_file

**Pull request merged**